charts/shoot-addons/charts/nginx-ingress/templates/clusterrole.yaml

```diff
@@ -12,24 +12,16 @@ rules:
   - apiGroups:
       - ""
     resources:
+{{- if semverCompare "< 1.20-0" .Capabilities.KubeVersion.GitVersion }}
       - configmaps
+{{- end }}
       - endpoints
       - nodes
       - pods
       - secrets
     verbs:
       - list
       - watch
-{{- if and .Values.controller.scope.enabled .Values.controller.scope.namespace }}
```

Comment:
Do you want to completely remove this option from the chart, i.e, `values.yaml` and `controller-deployment.yaml`?

Reply:
done